pilot/docker/Dockerfile.proxyv2

```diff
@@ -14,10 +14,19 @@ COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
 
 RUN chown -R istio-proxy /var/lib/istio
 
+COPY istio-iptables.sh /usr/local/bin/istio-iptables
+
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
 # hadolint ignore=DL3007
 FROM gcr.io/distroless/cc:latest as distroless
 
+COPY --from=default /sbin/xtables-multi /sbin/iptables* /sbin/ip6tables* /sbin/ip /sbin/
```

Comment:
That's the part I don't like about distroless, and I'm pretty worried this will create problems. 

This will override the 'distroless' lib with the libs from default.  Is the default on the same glibc as distroless ? 

( we do have this problem for envoy as well, but only in the sense that envoy build may be on an incompatible glibc and not work - we're not replacing/overriding .so files in the distroless base ).

Reply:
I don't like this either, but for what its worth this is not new in this PR, it is copied over from proxy_init.